spring-cloud-starter-task-spark-client/README.adoc

```diff
@@ -19,4 +19,16 @@ $$spark.resource-archives$$:: $$A comma separated list of archive files to be in
 $$spark.resource-files$$:: $$A comma separated list of files to be included with the application submission.$$ *($$String$$, default: `$$<none>$$`)*
 //end::configuration-properties[]
 
+== Example
```

Comment:
Can we add that `Build with Maven` entry like in the other app starters? I can add it when merging.